lib/permutations.rb

```diff
@@ -1,4 +1,29 @@
-
 def permutations?(string1, string2)
-  raise NotImplementedError, "permutations? not implemented"
-end
\ No newline at end of file
+
+  def str_to_hash(string)
```

Comment:
Mostly a style comment:  I don't like putting a method in a method in Ruby.  In JavaScript it's natural, not so much here.